### ISSUE TITLE
distribute with vim?

### ISSUE THREAD

**Issue by chrisbra** [nanotech/jellybeans.vim#69]
there is an [issue](https://github.com/vim/vim/issues/1665) open in the vim repository, to distribute more colorschemes with Vim. Would you kindly give your permission to distribute this colorscheme with Vim?

**Comment by nanotech** [comment]
Yes, that would be great! There's a few open issues I'd like to resolve first, then I'll tag a new release.

**Comment by chrisbra** [comment]
Thanks. Well, I can collect your file and send it to Bram, or you can do it. I would prefer if you can do it, since you know obviously best, when it is kind of stable enough and of course you will know when to send updated themes to Bram for inclusion. If you want to do it, mention the issue 1665 when sending it to Bram. Last Bram said, he would collect them and decide later what to bundle with Vim, so I cannot guarantee that it will be bundled later on.
